src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml

```diff
@@ -178,15 +178,85 @@
 
 	<!--#region Default Material NavigationBar-->
 	<Style x:Key="MaterialMainCommandStyle"
-		   BasedOn="{StaticResource MaterialAppBarButtonStyle}"
-		   TargetType="AppBarButton">
-		<Setter Property="Foreground" Value="{ThemeResource NavigationBarMainCommandForeground}" />
-		<!-- Native back icons are used on mobile platforms -->
-		<not_mobile:Setter Property="Icon" win:Value="{x:Null}">
-			<not_win:Setter.Value>
-				<PathIcon Data="{ThemeResource NavigationBarBackIconData}" />
-			</not_win:Setter.Value>
-		</not_mobile:Setter>
+       TargetType="AppBarButton">
+		<Setter Property="Background" Value="{ThemeResource AppBarButtonBackground}" />
+		<Setter Property="Foreground" Value="{ThemeResource AppBarButtonForeground}" />
+		<Setter Property="BorderBrush" Value="{ThemeResource AppBarButtonBorderBrush}" />
+		<Setter Property="HorizontalAlignment" Value="Left" />
+		<Setter Property="VerticalAlignment" Value="Top" />
+		<Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+		<Setter Property="FontWeight" Value="Normal" />
+		<Setter Property="Width" Value="68" />
+		<Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+		<Setter Property="FocusVisualMargin" Value="-3" />
+		<Setter Property="AllowFocusOnInteraction" Value="False" />
+		<Setter Property="Content" Value="{ThemeResource NavigationBarBackIconData}" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="AppBarButton">
+					<um:Ripple Feedback="{TemplateBinding Foreground}"
+                           FeedbackOpacity="{StaticResource PressedOpacity}"
+                           MinWidth="{TemplateBinding MinWidth}"
+                           MaxWidth="{TemplateBinding MaxWidth}">
+						<Grid x:Name="Root"
+                          Background="Transparent">
```

Comment:
Let's fixup the whitespace for this entire file so it is all tabs:

![image](https://github.com/user-attachments/assets/9570df85-3e98-4acc-b6b5-e45751760597)